app/client/src/features/events/Questions/AskQuestion/AskQuestion.tsx

```diff
@@ -37,20 +38,36 @@ export const ASK_QUESTION_MUTATION = graphql`
     }
 `;
 
+function isURL(str: string) {
+    const pattern = new RegExp('^(https?:\\/\\/)?'+
+      '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|'+ 
+      '((\\d{1,3}\\.){3}\\d{1,3}))'+ 
+      '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*'+ 
+      '(\\?[;&a-z\\d%_.~+=-]*)?'+ 
+      '(\\#[-a-z\\d_]*)?$','i'); 
+    return !!pattern.test(str);
+}
+
```

Comment:
I would move this to `src/utils/index.ts`.  We may have other places where this could be used in the future.